lang/cs/Org.Apache.REEF.Common/Services/ServiceConfiguration.cs

```diff
@@ -62,6 +72,17 @@ public static ConfigurationModule ConfigurationModule
             }
         }
 
+        public static string WrapServiceConfigAsString(IConfiguration serviceConfiguration)
```

Comment:
The method is used cross modules so it must be public.  Another option is put it inside AllocatedEvaluator class as a private method. I feel put it in this class is more relevant because the code to do wrapping and unwrapping are in the same class. It is easy for maintenance.